test.js

```diff
@@ -129,7 +139,7 @@ it('normal throw', function() {
     'throw new Error("test");'
   ], [
     'Error: test',
-    /^    at Object\.exports\.test \(.*\/line1\.js:1001:101\)$/
+    /^    at Object\.exports\.test \((?:.*\/)?line1\.js:1001:101\)$/
```

Comment:
I also think it's a good idea to have stricter tests, so I'm leaning against updating `compareStackTrace` and have 2 regexps.